docs/DeveloperGuide.md

```diff
@@ -650,7 +623,7 @@ Those without any stars are user stories that were considered but will not be im
 
 
 
-**Use case: UCSH04 Receive a prompt when clearing the system**
+### **Use case: UCSH05 — Receive a prompt when clearing the system**
```

Comment:
Similarly, do we have this? we don't right (i just tried), then we could remove it?

Reply:
This was in the original DeveloperGuide. I also thought of deleting it but wasn't sure as it was part of our Use Cases google sheets